src/sentry/eventstream/transactions/backend.py

```diff
@@ -0,0 +1,135 @@
+import logging
+from dataclasses import dataclass
+from typing import Any, Literal, Mapping, Union
+
+from django.conf import settings
+
+from sentry import quotas
+from sentry.eventstream.abstract import EventStreamAPI
+from sentry.eventstream.backends import KafkaEventStreamBackend, SnubaEventStreamBackend
+from sentry.eventstream.utils import (
+    dispatch_post_process_group_task,
+    encode_bool,
+    get_unexpected_tags,
+    strip_none_values,
+)
+from sentry.utils.sdk import set_current_event_project
+from sentry.utils.services import Service
+
+logger = logging.getLogger(__name__)
+
+from sentry.eventstream.kafka.postprocessworker import TransactionsPostProcessForwarderWorker
+
+
+@dataclass
+class TransactionsInsertData:
+    event: Any
+    received_timestamp: float
+    skip_consume: bool
+
+
+class TransactionsEventStreamAPI(EventStreamAPI, Service):
+    __all__ = ("insert",)
+
+    def __init__(self):
+        self._backend = None
+
+    def run_post_process_forwarder(
```

Comment:
Looking at the `run_post_process_forwarder` method I feel it does not need to exist in the API class at all. 
IT does nothing other than delegate the call to the backend in both implementations.
Removing it will also help you with the decision whether to keep an abstract API class or not.